internal/serve/httphandler/verifiy_receiver_registration_handler.go

```diff
@@ -253,7 +277,7 @@ func (v VerifyReceiverRegistrationHandler) VerifyReceiverRegistration(w http.Res
 		}
 
 		// STEP 5: PATCH transaction on the AnchorPlatform
```

Comment:
We can add to this comment that we also update the `AP Transaction ID` on the `receiver_wallets` table.

Reply:
Addresed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/44/commits/9876661a08d58850949a2c00bfa6d428f627b7d5